packages/typespec-ts/test/modularIntegration/extensibleEnums.spec.ts

```diff
@@ -41,7 +41,7 @@ describe("ExtensibleEnums Rest Client", () => {
 
   it("should put unknown value and receives 500", async () => {
     try {
-      const result = await client.string.putUnknownValue("Weekend", {
+      const result = await client.string.putUnknownValue("Weekend" as any, {
```

Comment:
In this case, we didn't enable the experimental extensible enums feature. so we will generate literal unions for this case, and unknown value will need to be casted.